erika-goodwin/Folder-1
# Add global menu and make each project own layouts

to help you having multiple projects at once i made couple changes

- main menu to go to your projects (insta and apple)
- layouts for your existing project. so if you want to make more instagram pages you can ;)
- added brand icons ;) you can use real instagram, apple and many other brand's icons

To test it : 
change the branche to this one (Add global menu and make each project own layouts)
then do 'npm i' in your console to install brands icons
if you want the change to your project you can push "Merge pull request" button bellow, this will apply my change to your project. (master) else it's a version by it's own in your project.
You can see the changes i made by clicking here (see red rectangle in image below)

![Capture d’écran 2019-11-24 à 21 56 10](https://user-images.githubusercontent.com/8041094/69495010-4f6cb980-0f05-11ea-9dee-089d26ecd171.png)


let me know if you have any question !! cya 

here a preview
![Enregistrement de l’écran 2019-11-24 à 21 40 39 mov](https://user-images.githubusercontent.com/8041094/69494919-62cb5500-0f04-11ea-80e8-49ffc7b3ab3b.gif)
